Settings.h

```diff
@@ -26,6 +28,7 @@ typedef struct {
 
 typedef struct Settings_ {
    char* filename;
+   int config_version;
```

Comment:
"unsigned int" here?  or size_t?  :)

Reply:
No config file reader is worth it, if it cannot handle -1 :-)